src/LogOutputManager.ts

```diff
@@ -327,7 +327,7 @@ export class LogOutputManager {
 
     public getMethodName(path: string, lineNumber: number, isFilePath: boolean): string | null {
         let fsPath = isFilePath ? path : this.docLinkProvider.convertPkgPathToFsPath(path);
-        const method = this.declarationProvider.getFunctionBeforeLine(fsPath, lineNumber);
+        const method = fsPath && this.declarationProvider.getFunctionBeforeLine(fsPath, lineNumber);
```

Comment:
This boolean-ish logic doesn't make much sense to me with respect to what is happening here. Do you think you could refactor it a little bit (perhaps an `if` statement) to make it more clear what role `fsPath` has to play in computing `method`.